install-wings.sh

```diff
@@ -220,15 +225,19 @@ function check_os_comp {
   fi
 
   virt_serv=$(virt-what)
-  if [ "$virt_serv" != "" ]; then
-    print_warning "Virtualization: ${virt_serv//$'\n'/ } detected."
-  fi
 
-  if [ "$virt_serv" == "openvz" ] || [ "$virt_serv" == "lxc" ] ; then # add more virtualization types which are not supported
-    print_warning "Unsupported type of virtualization detected. Please consult with your hosting provider whether your server can run Docker or not. Proceed at your own risk."
-    print_error "Installation aborted!"
-    exit 1
-  fi
+  case "$virt_serv" in
+    openvz | lxc)
+      print_warning "Unsupported type of virtualization detected. Please consult with your hosting provider whether your server can run Docker or not. Proceed at your own risk."
+      print_error "Installation aborted!"
+      exit 1
+      ;;
+    "")
+      ;;
```

Comment:
Really nit-picking here, so disregard this as you like (since this PR is amazing as it is and this is by far better than it already is), but can you do this without catching the empty case? I understand that if `virt_serv` is empty it shouldn't do anything at all so this statement looks redundant to me? Perhaps I am missing something?

Reply:
So this is here, because it should give an error when the `virt_serv` string is empty. I actually searched for a better solution, but apart from regex, couldn't find anything. Going back to `if` might also be an option.

Reply:
Sorry, I am not too sure what you mean. `virt_serv` is empty when there is no virtualization detected, which is quite common for a lot of people running Wings. Right now, the code will not print anything when `virt_serv` is empty but it will print the warning below if it's not empty nor handled by the case above. If you think it should give an error when `virt_serv` is empty, that is not what the code is doing currently.

Reply:
Ok first of all I had a typo there and instead of 'shouldn't' I wrote 'should'. 

Next thing is that there isn't any other solution when using `case`, because it needs to print warning when `lxc` or `openvz` and it needs to print warning when that string is not empty. If I remove that `"")` it will print the warning without any virtualization inside of it. Of course a simple check for an empty string at the start of `print_warning`  would be viable. Or as I said before a fallback to `if` is also viable.

Reply:
It looks to me like you solved it in another way! :+1:

Reply:
Yeah just was tired of overthinking this.